### PR TITLE
docs(readme): we are not mongoDB.

### PR DESCRIPTION
**Pull request by fncolon**
fix funny typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ MangoDB is an open-source proxy, which converts MongoDB wire protocol queries to
 
 ## Why do we need MangoDB?
 
-MongoDB is a life-changing technology for many developers, empowering them to build applications faster than using relational databases.
+MangoDB is a life-changing technology for many developers, empowering them to build applications faster than using relational databases.
```
[the remark this answers is not in the document]
Comment (fncolon) [reply README.md:9]: but why?

[the remark this answers is not in the document]
Comment (fncolon) [reply README.md:9]: but why??

[the remark this answers is not in the document]
Comment (fncolon) [reply README.md:9]: Most `MongoDB` users are not in need of many of the advanced features offered by `MongoDB`; however, they are in need of an open-source database solution. Recognizing this, MangoDB is here to fill the gap by providing an alternative.
what do you mean by versusing the same thing. KEKW

[the remark this answers is not in the document]
Comment (fncolon) [reply README.md:9]: Oh i see, can you resolve the conversation? i won't

[the remark this answers is not in the document]
Comment (ii64) [reply README.md:9]: Loved it ~~MongoDB~~ MangoDB

**Pull request closed without merging**